app/src/main/java/eu/kanade/presentation/manga/components/MangaDialogs.kt

```diff
@@ -117,17 +117,19 @@ fun SetIntervalDialog(
                         contentAlignment = Alignment.Center,
                     ) {
                         val size = DpSize(width = maxWidth / 2, height = 128.dp)
-                        val items = (0..FetchInterval.MAX_INTERVAL)
+                        val items = (-1..FetchInterval.MAX_INTERVAL)
```

Comment:
You can create a list with -1 at the end like this: ((0..FetchInterval.MAX_INTERVAL) + listOf(-1))

Reply:
Another option is add a LongClick, when user long click the timeglass then set update interval to -1. Remember to show a toast message.

Reply:
> You can create a list with -1 at the end like this: `(0..FetchInterval.MAX_INTERVAL) + listOf(-1))`

I'll try that out

> Another option is add a LongClick, when user long click the timeglass then set update interval to -1. Remember to show a toast message.

This is something that would be really cool to add tbh. I'll look into it

Reply:
If I do that, it starts having other errors through the whole project